doc/content/getting-started/migrating-from-v2/_index.md

```diff
@@ -4,4 +4,4 @@ description: ""
 weight: -1
 ---
 
-This guide documents the process of migrating end devices from {{% ttnv2 %}} to {{% tts %}}  in bulk, without the need for devices to initiate a rejoin.
+This guide documents the process of migrating end devices from {{% ttnv2 %}} to {{% tts %}}.
```

Comment:
Isn't this important? @laurensslats

Reply:
Yeah it is, I removed it just because it's already said somewhere in the overview of the Migrating section, but it's fine, it's easier to keep it and avoid confusion